README.md

```diff
@@ -11,66 +11,122 @@ pages with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/).
 
 ## How to do a Quick Edit on github
 
-To do a quick edit, use this is a 3 step process:
+To do a quick edit, use this is a 4 step process:
 
-1st: Make your edit and commit to the develop branch
+1st: Make your edit and commit to a new branch.
 
-2nd: Make a PR from develop to master, get that reviewed
+2nd: Make a PR from your branch to develop, get that reviewed.
 
-3rd: Merge the PR to master
+3rd: Merge the PR to develop.
 
-This commits directly to the `develop` branch. If you are doing a more involved
-group of edits or changes, please use a process like git flow. If you are not
-familiar with this ask a dev or an admin to help you.
+4th: Create a PR to deploy your changes to master.
 
 ### 1st: Make your edit on github
-To edit a page first go to the info.arxiv.org page you want to edit. Ex https://info.arxiv.org/help/gzip.html
+- To edit a page first go to the info.arxiv.org page you want to edit. Ex https://info.arxiv.org/help/gzip.html
 
-![gzip page](non-info/help-info-page.png)
+- Click on the pencil icon to edit the page. That will take you to the corresponding page in github.
 
-There click on the pencil icon to edit the page. That will take you to the corresponding page in github.
+<kbd>![Screenshot of a page on info.arxiv.org and the location of the edit icon](/non-info/02-help-info-page.png)</kbd>
 
-![gzip page](non-info/help-github1.png)
+- On the page you want to edit in github, click the pencil to edit the page contents right in github.
 
-There click "edit" and you can edit contents of the page in github. Once you
-have made changes you can click preview to see how it will look when completed
-There are slight differences between the markdown we use and what
-github uses.
+<kbd>![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)</kbd>
 
-![gzip page](non-info/help-github2.png)
+- After you have made changes, click preview to see how it will look when completed. 
+**Please note:** Github uses "Github flavored markdown" which is different from the markdown we use to create our documentation. There may be small variations in how the markdown appears due to these differences.
 
-Once you are happy with your changes you can save the by going to the bottom of
-the edit and "Commit changes". Commit is the term github and git use for
-saving. You should put a brief note in the first text area. You may add longer
-note in the larger text area if you have more to say about your changes. Then
-you can commit your changes to the branch `develop`.
+<kbd>![Screenshot of the location of the preview button on the edited page in Github](/non-info/03-help-preview-github.png)</kbd>
```

Comment:
This screen shot should show what Preview mode looks like, not just show the button

Reply:
@cbf66 I revised the instructions and added a screenshot displaying the preview.